docs/user/alerting/rule-types/es-query.asciidoc

```diff
@@ -12,33 +12,41 @@ threshold condition is met.
 Fill in the <<defining-rules-general-details, rule details>>, then select 
 *{es} query*.
 
+[float]
+=== Select the query type
+
+{es} query rule can be defined using KQL/Lucene or Query DSL.
```

Comment:
IMO, this screenshot and section is unnecessary. I would recommend just putting this "{es} query rule can be defined..." sentence at the end of the paragraph in the previous section.

Reply:
Updated in https://github.com/elastic/kibana/pull/144689/commits/6893e1f958d28a59a6417d3cc2f8bb2006b2092f!